orders/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+
+class Order(models.Model):
+    total_quantity     =   models.IntegerField(null=True)
+    shipping_price     =   models.DecimalField(max_digits=7,decimal_places=2,null=True)
+    user               =   models.ForeignKey('users.User',on_delete = models.CASCADE)
+    total_price        =   models.DecimalField(max_digits=10,decimal_places=2,null=True)
+    status             =   models.ForeignKey('OrderStatus',on_delete = models.CASCADE)
+    payment_type       =   models.ForeignKey('PaymentType',on_delete = models.CASCADE,null=True)
```

Comment:
```suggestion
    payment_type       =   models.ForeignKey('PaymentType',on_delete = models.CASCADE,null=True)
```
전반적으로 CASCADE를 많이 사용하시는데, 이게 간편하지만, 나중에 데이터 조작시에 문제가 될 수 있습니다. 연관된 모든 데이터가 날아 갈수 있어서. SET_NULL, 권장드립니다. SET_NULL 하려면 null=True여야 합니다.

Reply:
SET_NULL은 고심끝에 꼭 삭제 되야하는 것들 빼고는 수정하였습니다. 리뷰 부탁드립니다.